src/pages/Models/Details/_model-details.scss

```diff
@@ -26,10 +26,16 @@
 
 @mixin model-details-header {
   .model-details__header {
-    align-items: center;
+    display: grid;
+    grid-template-columns: 1fr;
 
     @media (min-width: $breakpoint-medium) {
-      display: flex;
+      gap: 1rem;
+      grid-template-columns: 240px 1fr;
+    }
+
+    @media (min-width: $breakpoint-m-large) {
```

Comment:
I'm not sure we want to be creating custom breakpoints, we should investigate instead overriding the vanilla `breakpoint-x-large` instead.

Reply:
We don't want to override vanilla breakpoints that may break any components relying on that existing breakpoint value.